source/gui/settingsDialogs.py

```diff
@@ -713,21 +713,29 @@ def makeSettings(self, settingsSizer):
 		# Translators: The label for a setting in general settings to allow NVDA to start after logging onto
 		# Windows (if checked, NVDA will start automatically after logging into Windows; if not, user must
 		# start NVDA by pressing the shortcut key (CTRL+Alt+N by default).
-		self.startAfterLogonCheckBox = wx.CheckBox(self, label=_("St&art NVDA after Windows sign in"))
+		self.startAfterLogonCheckBox = wx.CheckBox(self, label=_("St&art NVDA after Windows sign-in for me"))
 		self.startAfterLogonCheckBox.SetValue(config.getStartAfterLogon())
 		if globalVars.appArgs.secure or not config.isInstalledCopy():
 			self.startAfterLogonCheckBox.Disable()
 		settingsSizerHelper.addItem(self.startAfterLogonCheckBox)
 
 		# Translators: The label for a setting in general settings to allow NVDA to come up in Windows login screen (useful if user needs to enter passwords or if multiple user accounts are present to allow user to choose the correct account).
-		self.startOnLogonScreenCheckBox = wx.CheckBox(self, label=_("Use NVDA on the Windows logon screen (requires administrator privileges)"))
+		self.startOnLogonScreenCheckBox = wx.CheckBox(
+			self,
+			label=_("Start NVDA before Windows sign-in for &everyone (requires administrator privileges)")

```

Comment:
I don't think "for everyone" is necessary. The sign in screen is inherently for everyone.
```suggestion
			label=_("Use NVDA at Windows sign-in (requires administrator privileges)")

```

I also would like to drop the "requires administrator privileges" part. It would be better to explain why it can't be done when trying. For now it's probably enough to explain in the user guide.